### ISSUE TITLE
New ego vehicles upload to wise fails

### ISSUE THREAD

**Issue by farukbaykara** [lgsvl/simulator#1981]
I'm trying to build a new ego vehicle, and the build is successfully completed in the unity editor. While uploading to the wise I get the following error
![Screenshot from 2022-02-08 17-58-42](https://user-images.githubusercontent.com/63922934/153015578-cf746a4d-374e-44d9-905f-710129b8fbb7.png)
Size of my AssetBundle is really big (550Mb). Can this cause this problem to occur? If this is the problem, how can I reduce the size of my AssetBundle?

**Comment by StepTurtle** [comment]
i had same problem,  and it was because my model was big over than 200 mb. Wise does not allow files over 200mb. You have to decrease your model size.

**Comment by EricBoiseLGSVL** [comment]
Yes, you need to remove texture data from the fbx (import separately in Unity) and reduce poly count of mesh
